### ISSUE TITLE
Plugin formatting fuctionn ol() doesn't work

### ISSUE THREAD

**Issue by Dual-0** [geluk/matrix-webhook-gateway#13]
Hi,

first of all: thanks for this amazing webhook gateway. The plugin building is great.
_No more E-Mail notifications about my infrastructure and or app updates._ Thanks for this!

Today I try the ol() function to create bullet point list but the log shows me this error:
```
2021-09-29 22:32:00.617.000         ERROR         [webhook-srv]                    Failed to handle webhook invocation:
Error         entries.map is not a function
```

Thanks

**Comment by geluk** [comment]
I'm glad you like it! Off the top of my head, there are two things that could be going wrong here:
 - The variable you're passing into `ol()` isn't an array. You can use `Array.isArray(value)` to check if `value` is an array.
 - The `map` function is not available in your version of JavaScript. This could happen if you're not running the application in Docker and you're using an older version of Node.js.

Could you check if one of these is the case?

**Comment by geluk** [comment]
Never mind, I found the issue. The way I've documented the `ol()` function in the readme isn't correct. It says:
```js
ol(a('https://example.com', 'first'), 'second', 'third')
```
But the function currently only accepts arrays, so it has to be called like this:
```js
ol([a('https://example.com', 'first'), 'second', 'third'])
```

In the next version I'll update the function to accept multiple arguments as well, but for now, making the change I've shown above should work.

**Comment by geluk** [comment]
Implemented in version 1.1.
